src/components/forms/formComponents.js

```diff
@@ -341,23 +359,57 @@ export const formInputRadioGroup = (config) => {
 export const formInputCheckbox = (config) => {
   const {
     id, disabled, error, toggleText,
-    formValue, ariaDescribedby
+    formValue, ariaDescribedby,
+    valueType, placeholder, setError
   } = config;
 
-  return div({ className: 'checkbox' }, [
+  return div({
+    className: 'checkbox',
+    style: {
+      margin: '1.5rem 0 1.5rem 0',
+    },
+  }, [
     input({
       type: 'checkbox',
-      id: `cb_${id}_${toggleText}`,
-      checked: formValue,
+      id: id,
+      checked: (valueType === 'string' ? isString(formValue) : formValue),
       className: 'checkbox-inline',
       'aria-describedby': ariaDescribedby,
-      onChange: (event) => onFormInputChange(config, event.target.checked),
+      onChange: (event) => {
```

Comment:
On my local instance I wasn't actually able to check any of the options under **Consent Secondary Data Use Terms**. I was digging around and saw that the EditConsentGroup uses FormFieldTypes.CHECKBOX which uses this formInputCheckbox components but I think that we may have it disabled for now, is that correct?